compiler/damlc/tests/daml-test-files/EmptyContractKeyMaintainers.daml

```diff
@@ -1,5 +1,6 @@
 -- @ERROR Attempt to create a contract key with an empty set of maintainers
--- @ERROR Attempt to fetch or lookup a contract key with an empty set of maintainers
+-- @ERROR Attempt to fetch, lookup or exercise a contract key with an empty set of maintainers
+-- @ERROR Attempt to fetch, lookup or exercise a contract key with an empty set of maintainers
```

Comment:
We should add the exact error locations here to make sure we get the errors where we expect them.

Reply:
Good to know.

Reply:
Thanks for adding them. 🙇